lib/instrumentation/ids.js

```diff
@@ -0,0 +1,31 @@
+const { stringify } = require('querystring')
+
+class Ids {
+  toString () {
+    return stringify(this, ',', '=')
+  }
+}
+
+class SpanIds extends Ids {
+  constructor (span) {
+    super()
+    this['trace.id'] = span.traceId
+    this['span.id'] = span.id
+    Object.freeze(this)
```

Comment:
Just out of curiosity: Why the freeze?

Reply:
No particular reason. There's no reason it _should_ be mutable, so I figured why not just make it _immutable_. 🤷‍♂